### ISSUE TITLE
dials.import does not allow to override the beam polarisation

### ISSUE THREAD

**Issue by dagewa** [dials/dials#592]
Beam geometry options are pretty limited:
```
beam_phil_scope = libtbx.phil.parse('''
  beam
    .expert_level = 1
    .short_caption = "Beam overrides"
  {

    wavelength = None
      .type = float
      .help = "Override the beam wavelength"

    direction = None
      .type = floats(size=3)
      .help = "Override the sample to source direction"
      .short_caption = "Sample to source direction"

  }
''')
```

**Comment by dagewa** [comment]
Strictly, a dxtbx issue, but the user stumbles across this first in `dials.import`

**Comment by jmp1985** [comment]
Looks like this was fixed in cctbx commit aa024ad9b6
